### PR TITLE
#4905 [Manager] File selector should support deletion

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4905 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Workspace/Settings/SettingsMainStyles.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import styled, { css } from 'styled-components';
 
 export const Wrapper = styled.div(
   ({ theme }) => css`
-    width: 680px;
+    width: 800px;
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Settings/SettingsMainStyles.js:21]: So the icons are not stacked?

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsMainStyles.js:21]: width: 680px
螢幕快照 2020-05-19 上午11 08 24

width: 800px
螢幕快照 2020-05-19 上午11 07 56

調整到 800px 看起來比較舒服

Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsMainStyles.js:21]: Got it 👍

**Comment by eechih** [comment]
retry manager-e2e

**Comment by eechih** [comment]
retry manager-e2e

**Comment by eechih** [comment]
retry manager-e2e

**Comment by eechih** [comment]
retry manager-e2e

**Comment by eechih** [comment]
retry manager-e2e

**Pull request merged**